foia_hub/tests/test_api.py

```diff
@@ -72,7 +82,10 @@ def test_reading_room_preparer(self):
         serialized_rooms = {'foia_libraries': [
             {'link_text': 'Url One', 'url': 'http://urlone.gov'},
             {'link_text': 'Url Two', 'url': 'http://urltwo.gov'}]}
-
+        if custom_backend == 'postgresql_psycopg2':
```

Comment:
Consider unittest's skipIf / skipUnless -- then the tests will explicitly output when elements are skipped